packages/cbioportal-utils/src/mutation/MutationUtils.spec.ts

```diff
@@ -164,6 +165,20 @@ describe('MutationUtils', () => {
                 'genomic location string should be:  1,100,100,C,G'
             );
         });
+        it('normalize chromosome and extract genomic location', () => {
```

Comment:
Thanks for adding a test! Apparently there is no test for the existing function `IGVUtils.normalizeChromosome`. So, at least the test is not duplicate :)

By the way, I think this should read:
```suggestion
        it('normalizes chromosome and extracts genomic location', () => {
```